src/Tracking/README.md

```diff
@@ -284,3 +284,54 @@ $this->conditionally_share_with_tags( Loaded::class );
 ```
 
  /Dev Info -->
+
+<woo-tracking-jsdoc>
+<!---
+Everything below will be automatically generated by `woo-tracking-jsdoc`.
+Do not edit it manually!
+-->
+
+### [`gla_ces_feedback`](js/src/components/customer-effort-score-prompt/index.js#L29)
+CES feedback recorded
+#### Emitters
+- [`CustomerEffortScorePrompt`](js/src/components/customer-effort-score-prompt/index.js#L49) whenever the CES feedback is recorded
+
+### [`gla_ces_modal_open`](js/src/components/customer-effort-score-prompt/index.js#L24)
+CES modal open
+#### Emitters
+- [`CustomerEffortScorePrompt`](js/src/components/customer-effort-score-prompt/index.js#L49) whenever the CES modal is open
+
+### [`gla_ces_snackbar_closed`](js/src/components/customer-effort-score-prompt/index.js#L19)
+CES prompt snackbar closed
+#### Emitters
+- [`CustomerEffortScorePrompt`](js/src/components/customer-effort-score-prompt/index.js#L49) whenever the CES snackbar (notice) is closed
+
+### [`gla_ces_snackbar_open`](js/src/components/customer-effort-score-prompt/index.js#L14)
+CES prompt snackbar open
+#### Emitters
+- [`CustomerEffortScorePrompt`](js/src/components/customer-effort-score-prompt/index.js#L49) whenever the CES snackbar (notice) is open
+
+### [`gla_edit_mc_store_address`](js/src/components/contact-information/store-address-card.js#L124)
+"Edit MC store address" Tracking event
+#### Properties
+|   |   |   |
+|---|---|---|
+`path` | `string` | A page from which the link was clicked.
+`subpath` | `string\|undefined` | A subpage from which the link was clicked.
+#### Emitters
+- [`StoreAddressCardPreview`](js/src/components/contact-information/store-address-card.js#L144) Whenever "Edit" is clicked.
+
+### [`gla_edit_wc_store_address`](js/src/components/contact-information/store-address-card.js#L23)
+"Edit WC store address" Tracking event
+#### Properties
+|   |   |   |
```

Comment:
💅 / ❓ 

The table shows an empty header. I will ad headers (like name, type, description) or remove that header. 

<img width="332" alt="Screenshot 2022-04-06 at 10 50 30" src="https://user-images.githubusercontent.com/5908855/161918722-452deaf3-1730-40bc-9770-eef778bea2f5.png">

Reply:
:til: (row as required prop)